README.md

```diff
@@ -40,10 +40,11 @@ To download and install the library and the binary, set up a Golang build enviro
 $ go get -u github.com/openshift/imagebuilder/cmd/imagebuilder
 ```
 
-The included command line takes two arguments, a path to a directory containing a dockerfile, and the name to tag the output image with.
+The included command line takes one argument, a path to a directory containing a Dockerfile. The `-t` option
+can be used to specify an image to tag as.
 
 ```
-$ imagebuilder path_to_directory output_image_name
+$ imagebuilder [-t TAG] DIRECTORY
```

Comment:
Isn't this a breaking change? What's our compatibility statement with respect to use of `imagebuilder` in scripts?